onnxruntime/core/mlas/lib/platform.cpp

```diff
@@ -171,6 +171,14 @@ MlasInitAMX()
 
 #endif // MLAS_TARGET_AMD64_IX86
 
+static inline bool ort_use_int8mm() {
```

Comment:
consider changing it to a compilation flag?

Reply:
Hi @chenfucn , do you mean make it compile time flag and enable the feature by default if the HW supports the I8MM instructions?